.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "bracketSameLine": true,
-  "printWidth": 150
-}
\ No newline at end of file
+  "printWidth": 150,
+  "singleQuote": true,
```

Comment:
I opted for single quotes, if you disagree I we can easily change it back. I just personally slightly prefer them

Reply:
nah, all good with me!